gateway-service/src/main/java/com/ca/mfaas/gateway/ws/WebSocketProxyServerHandler.java

```diff
@@ -58,6 +58,12 @@ public void addRoutedServices(String serviceId, RoutedServices routedServices) {
         routedServicesMap.put(serviceId, routedServices);
     }
 
+    private String getTargetUrl(String serviceUrl, ServiceInstance serviceInstance, String path) {
+        String servicePath = serviceUrl.charAt(serviceUrl.length() - 1) == '/' ? serviceUrl : serviceUrl + "/";
```

Comment:
This line is causing SonarQube minor issue, squid:S1075. Please feel free to suppress it.

I would suggest to use Sinarlint, if you are not using it already.